infra-parent/infra-v2/src/main/java/com/redhat/service/smartevents/infra/v2/api/models/ConditionStatus.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.service.smartevents.infra.v2.api.models;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.redhat.service.smartevents.infra.core.models.BaseEnumeration;
+
+public enum ConditionStatus implements BaseEnumeration {
```

Comment:
There have been issues with Jackson serialisation vs what we advertise in OpenAPI.

This approach fixes it (and is what we do on other enums).